packages/config/config/devices/0x0109/zp3113-7.json

```diff
@@ -0,0 +1,143 @@
+// Vision Security ZP3113-7
+// 4 in 1 PIR Sensor
+{
+	"manufacturer": "Vision Security",
+	"manufacturerId": "0x0109",
+	"label": "ZP3113-7",
+	"description": "4 in 1 PIR Sensor",
+	"devices": [
+		{
+			"productType": "0x2021",
+			"productId": "0x2112",
+			"zwaveAllianceId": 3806
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Temperature Units",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "°C",
+					"value": 0
+				},
+				{
+					"label": "°F",
+					"value": 1
+				}
+			]
+		},
+		"2": {
+			"label": "Temperature Reporting Interval",
+      "description: "0.1°C~5°C",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 50,
+			"defaultValue": 30,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Humidity Reporting Interval",
+      "description: "1%~50%",
+			"valueSize": 1,
```

Comment:
Make the unit "%" Then you can lose the range

Reply:
If so, a description would be nice